bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -2,68 +2,73 @@
 
 #NoLocalizationPlayerBehavior
 $BallSeen
     YES --> $BallClose
         YES --> $GoalSeen
             YES --> #GoAndKickBallDetectionGoal
             NO --> #GoAndKickBallAway
-        NO --> @GoToBall + target:close
-    NO --> @Stop, @LookAtBall, @SearchBall
+        NO --> @ChangeAction + action:going_to_ball, @GoToBall + target:close
+    NO --> @ChangeAction + action:searching, @Stop, @LookAtBall, @SearchBall
 
 #NoLocalizationGoalieBehavior
 $BallSeen
     YES --> $BallClose
         YES --> #GoAndKickBallAway
-        NO --> @Stop, @LookAtBall
-    NO --> @Stop, @LookAtBall
+        NO --> @ChangeAction + action:waiting, @Stop, @LookAtBall
+    NO --> @ChangeAction + action:searching, @Stop, @SearchBall
 
 #Positioning
 $LocalizationAvailable
-    YES --> @LookAtFieldFeatures, @StandAndWait + duration:%behavior/body/ready_wait_time, @AvoidBallActive, @GoToRolePosition
-    NO --> @GoToRelativePosition + x:2 + y:0 + t:0
+    YES --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @StandAndWait + duration:%behavior/body/ready_wait_time, @AvoidBallActive, @GoToRolePosition
+    NO --> @ChangeAction + action:positioning, @GoToRelativePosition + x:2 + y:0 + t:0
 
 #PositioningReady
 $LocalizationAvailable
-    YES --> @LookAtFieldFeatures, @StandAndWait + duration:%behavior/body/ready_wait_time, @AvoidBallActive, @GoToRolePosition
-    NO --> @GoToRelativePosition + x:2 + y:0 + t:0, @StandAndWait
+    YES --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @StandAndWait + duration:%behavior/body/ready_wait_time, @AvoidBallActive, @GoToRolePosition
+    NO --> @ChangeAction + action:positioning, @GoToRelativePosition + x:2 + y:0 + t:0, @StandAndWait
+
+#ReorientationPause
+$TimerRunning + name:reorientation
+    YES --> $TimerEnded + name:reorientation
+        YES --> @StartTimer + name:reorientation_pause + duration:%behavior/body/reorientation_pause_duration
+        NO --> @ChangeAction + action:localizing, @LookAtFieldFeatures, @Stop
+    NO --> @StartTimer + name:reorientation + duration:%behavior/body/reorientation_duration
 
 #GoalieBehavior
 $LocalizationAvailable
     YES --> $LocalizationPrecision
         HIGH --> $TimerRunning + name:reorientation
             YES --> @EndTimer + name:reorientation, @StartTimer + name:reorientation_pause + duration:%behavior/body/reorientation_pause_duration
             NO --> $BallSeen
-                YES --> $BallDangerous
-                    NO --> @GoToBlockPosition, @LookAtFieldFeatures
-                    ELSE --> #GoAndKickBallMapGoal
-                NO --> @LookAtFieldFeatures
+                YES --> $ClosestToBall
+                    YES --> $BallDangerous
+                        NO --> @ChangeAction + action:positioning, @GoToBlockPosition, @LookAtFieldFeatures
+                        ELSE --> #GoAndKickBallMapGoal
+                    NO --> @ChangeAction + action:going_to_ball, @GoToBlockPosition, @LookAtFieldFeatures
```

Comment:
Why is `@GoToBlockPosition, @LookAtFieldFeatures` both `action:positioning` and `@ChangeAction + action:going_to_ball`?

Reply:
that was wrong